### PR TITLE
added challenge question for user to retrieve the elb URL

### PR DESCRIPTION
**Pull request by vjsikha**
*Issue #, if available:*

#157 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by vjsikha** [comment]
@brentley launch button in https://eksworkshop.com/codepipeline/codepipeline/ opens the URL https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?stackName=eksws-codepipeline&templateURL=https://s3.amazonaws.com/eksworkshop.com/templates/master/ci-cd-codepipeline.cfn.yml 

The above is failing and it should be updated as below:
https://console.aws.amazon.com/cloudformation/home?#/stacks/create/review?stackName=eksws-codepipeline&templateURL=https://s3.amazonaws.com/eksworkshop.com/templates/master/ci-cd-codepipeline.cfn.yml

please let me know the best option to update "{{% cf-launch "ci-cd-codepipeline.cfn.yml?stackName=eksws-codepipeline" %}}"

File: content/codepipeline/change.md [hunk 1]
```diff
@@ -29,14 +29,17 @@ After you modify and commit your change in GitHub, in approximately one minute y
 
 ![CodePiepline Running](/images/codepipeline/codepipeline_building.png)
 
-Once the service is built and delivered, run the following command to get the Elastic Load Balancer (ELB) endpoint and open it in a browser.
-If the message is not updated immediately, give Kubernetes some time to deploy the change.
+#### Challenge:
+**How can we validate the changes and access latest version of the app?**
+ {{%expand "Expand here to see the solution" %}}
```
Comment (dalbhanj) [inline content/codepipeline/change.md]: I would add a HINT
HINT: Which command will get you Elastic Load Balancer (ELB) endpoint for this app?

File: content/codepipeline/codepipeline.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Click the **Launch** button to create the CloudFormation stack in the AWS Manage
 
 | Launch template |  |  |
 | ------ |:------:|:--------:|
-| CodePipeline & EKS |  {{% cf-launch "ci-cd-codepipeline.cfn.yml" "eksws-codepipeline" %}} | {{% cf-download "ci-cd-codepipeline.cfn.yml" %}}  |
+| CodePipeline & EKS |  {{% cf-launch "eksws-codepipeline" "ci-cd-codepipeline.cfn.yml" %}} | {{% cf-download "ci-cd-codepipeline.cfn.yml" %}}  |
```
Comment (dalbhanj) [inline content/codepipeline/codepipeline.md]: Can you rebase with upstream? looks like you don't have latest changes

File: layouts/shortcodes/cf-launch.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <a
-  href="https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?stackName={{ .Get 1 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 0 }}"
+href="https://console.aws.amazon.com/cloudformation/home?#/stacks/create/review?stackName={{ .Get 0 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 1 }}"
```
Comment (dalbhanj) [inline layouts/shortcodes/cf-launch.html]: Can you rebase with upstream? looks like you don't have latest changes

File: content/codepipeline/change.md (hunk 1 above)
Comment (vjsikha) [reply content/codepipeline/change.md]: added and pushed

File: content/codepipeline/codepipeline.md (hunk 2 above)
Comment (vjsikha) [reply content/codepipeline/codepipeline.md]: done

File: layouts/shortcodes/cf-launch.html (hunk 3 above)
Comment (vjsikha) [reply layouts/shortcodes/cf-launch.html]: done

**Comment by dalbhanj** [comment]
@vjsikha still has commits to cicd chapter. Please remove these changes and send commits to health checks chapter only

/content/codepipeline/codepipeline.md
/layouts/shortcodes/cf-launch.html

**Comment by oak2278** [comment]
How do we feel about moving that challenge to the page before? We do the same step to get the ELB so it feels more natural to put it there.

File: layouts/shortcodes/cf-launch.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
```
Comment (brentley) [inline layouts/shortcodes/cf-launch.html]: I believe this file is part of an earlier version of the codebase, and this diff is reverting back to the older version. Please remove it from this PR.

Comment (vjsikha) [reply layouts/shortcodes/cf-launch.html]: agreed. confirmed that it's an issue when user is on old version of Cloud Formation console. We will convey that all users must be on the newer version. reverting these changes

**Comment by dalbhanj** [comment]
should we close this PR in favor of https://github.com/aws-samples/eks-workshop/pull/172

File: content/codepipeline/codepipeline.md (hunk 2 above)
Comment (brentley) [inline content/codepipeline/codepipeline.md]: not sure if this also needs to be reverted since reverting the commit in cf-launch?

**Pull request closed without merging**